src/test/java/com/fasterxml/jackson/failing/read/LocationOfError1173Test.java

```diff
@@ -0,0 +1,326 @@
+            10,
+            1,
+            11
+        ),
+        new InvalidJson(
+            "Missing comma between object properties",
+            "{\"key1\":\"value1\" \"key2\":\"value2\"}",
+            17,
+            17,
+            1,
+            18
+        ),
+        new InvalidJson(
+            "Number as a property key",
+            "{1234: 5678}",
+            1,
+            1,
+            1,
+            2
+        ),
+        new InvalidJson(
+            "false literal as property key",
+            "{false: true}",
+            1,
+            1,
+            1,
+            2
+        ),
+        new InvalidJson(
+            "true literal as property key",
+            "{true: false}",
+            1,
+            1,
+            1,
+            2
+        ),
+        new InvalidJson(
+            "null literal as property key",
+            "{null: \"content\"}",
+            1,
+            1,
+            1,
+            2
+        ),
+        new InvalidJson(
+            "Missing comma between list elements",
+            "[\"no\" \"commas\"]",
+            6,
+            6,
+            1,
+            7
+        ),
+        new InvalidJson(
+            "Property key/value delimiter in list",
+            "[\"still\":\"invalid\"]",
+            8,
+            8,
+            1,
+            9
+        ),
+        new InvalidJson(
+            "Unexpected EOF",
+            "{",
+            1,
+            1,
+            1,
+            2
+        ),
+        new InvalidJson(
+            "Close marker without matching open marker",
+            "}",
+            0,
+            0,
+            1,
+            1
+        ),
+        new InvalidJson(
+            "Mismatched open/close tokens",
+            "{\"open\":\"close\"]",
+            15,
+            15,
+            1,
+            16
+        ),
+        new InvalidJson(
+            "Bare strings in JSON",
+            "{missing: quotes}",
+            1,
+            1,
+            1,
+            2
+        ),
+        new InvalidJson(
+            "Incorrect case for false literal",
+            "{\"isThisValidJson\": FALSE}",
+            24,
+            24,
+            1,
+            25
+        ),
+        new InvalidJson(
+            "Incorrect case for true literal",
+            "{\"shouldYouAvoidWritingJsonLikeThis\": TRUE}",
+            41,
+            41,
+            1,
+            42
+        ),
+        new InvalidJson(
+            "Incorrect case for null literal",
+            "{\"licensePlate\": NULL}",
+            20,
+            20,
+            1,
+            21
+        ),
+        new InvalidJson(
+            "Invalid JSON with raw unicode character",
+            // javac will parse the unicode control sequence, it will be passed to the parser as a raw unicode character
+            "{\"validJson\":\"\u274c\",\"right\", \"here\"}",
+            26,
+            24,
+            1,
+            25
```

Comment:
Parser backends aren't consistent about the column in this case, because the exact definition of "column" seems to depend on whether the parser is operating on a byte array or a character array. I'm open to suggestions about what to do with this case.

Reply:
Column should be exactly same for byte/char -backed input, for ASCII characters (for multi-byte characters there is indeed difference). But I think few tests use characters outside ASCII range..